.github/fabricbot.json

```diff
@@ -1,939 +0,0 @@
-        "actions": [
-          {
-            "name": "addLabel",
-            "parameters": {
-              "label": "approved"
-            }
-          },
-          {
-            "name": "removeLabel",
-            "parameters": {
-              "label": "new"
-            }
-          },
-          {
-            "name": "addLabel",
-            "parameters": {
-              "label": "help wanted"
-            }
-          }
-        ]
-      }
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssuesOnlyResponder",
-      "version": "1.0",
-      "id": "x-cqRQZU3d3zC29TPLutp",
-      "config": {
-        "conditions": {
-          "operator": "and",
-          "operands": [
-            {
-              "name": "addedToProjectColumn",
-              "parameters": {
-                "projectName": "New Feature Proposals",
-                "columnName": "Proposal Championed"
-              }
-            },
-            {
-              "name": "addedToProjectColumn",
-              "parameters": {
-                "projectName": "New Feature Proposals",
-                "columnName": "Proposal Approved"
-              }
-            },
-            {
-              "name": "addedToProjectColumn",
-              "parameters": {
-                "projectName": "New Feature Proposals",
-                "columnName": "Pull Request Approved (Pending Documentation)"
-              }
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Add Pending Documentation Label",
-        "actions": [
-          {
-            "name": "addLabel",
-            "parameters": {
-              "label": "pending documentation"
-            }
-          },
-          {
-            "name": "removeLabel",
-            "parameters": {
-              "label": "new"
-            }
-          }
-        ]
-      }
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssuesOnlyResponder",
-      "version": "1.0",
-      "id": "hGFY_RbEjygXzEejNBsUK",
-      "config": {
-        "conditions": {
-          "operator": "or",
-          "operands": [
-            {
-              "name": "addedToProjectColumn",
-              "parameters": {
-                "projectName": "New Feature Proposals",
-                "columnName": "Documentation Approved"
-              }
-            },
-            {
-              "name": "addedToProjectColumn",
-              "parameters": {
-                "projectName": "New Feature Proposals",
-                "columnName": "Completed"
-              }
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Add Documentation Approved Label",
```

Comment:
Project board automation is not supported in the new service, so related tasks were not ported.